tests/unit/ui/components/TestUserRating.js

```diff
@@ -0,0 +1,82 @@
+import * as React from 'react';
+
+import { denormalizeReview } from 'amo/actions/reviews';
+import UserRating, { UserRatingBase } from 'ui/components/UserRating';
+import { logOutUser } from 'amo/reducers/users';
+import {
+  dispatchClientMetadata,
+  dispatchSignInActions,
+  fakeReview,
+} from 'tests/unit/amo/helpers';
+import {
+  fakeI18n,
+  shallowUntilTarget,
+} from 'tests/unit/helpers';
+
+let store;
+
+beforeEach(() => {
+  store = dispatchClientMetadata().store;
+});
+
+function render(customProps = {}) {
+  const props = {
+    i18n: fakeI18n(),
+    store,
+    ...customProps,
+  };
+  return shallowUntilTarget(<UserRating {...props} />, UserRatingBase);
+}
+
+function signInAndReturnReview({ siteUserId, reviewUserId }) {
+  dispatchSignInActions({ store, userId: siteUserId });
+  return denormalizeReview({
+    ...fakeReview,
+    user: {
+      ...fakeReview.user,
+      id: reviewUserId,
+    },
+  });
+}
+
+describe('ui/components/UserRating', () => {
+  it('renders a Rating', () => {
+    const props = {
+      review: fakeReview,
+      className: 'my-class',
+      readOnly: true,
+      styleSize: 'small',
+    };
+    const root = render(props);
```

Comment:
I'm not really sure how helpful this test is. Maybe it would be more helpful like this?

```diff
diff --git a/tests/unit/ui/components/TestUserRating.js b/tests/unit/ui/components/TestUserRating.js
index 9bea9a1e8..ddfd2aa1a 100644
--- a/tests/unit/ui/components/TestUserRating.js
+++ b/tests/unit/ui/components/TestUserRating.js
@@ -1,6 +1,7 @@
 import * as React from 'react';
 
 import { denormalizeReview } from 'amo/actions/reviews';
+import Rating from 'ui/components/Rating';
 import UserRating, { UserRatingBase } from 'ui/components/UserRating';
 import { logOutUser } from 'amo/reducers/users';
 import {
@@ -47,7 +48,7 @@ describe('ui/components/UserRating', () => {
       readOnly: true,
       styleSize: 'small',
     };
-    const root = render(props);
+    const root = render(props).find(Rating);
     expect(root).toHaveProp('className', props.className);
     expect(root).toHaveProp('readOnly', props.readOnly);
     expect(root).toHaveProp('styleSize', props.styleSize);
```